src/app/archive_hardfork_toolbox/tests/test_convert_canonical.ml

```diff
@@ -0,0 +1,351 @@
+open Core
+open Async
+open Archive_hardfork_toolbox_lib
+open Caqti_request.Infix
+
+(* Test database setup and teardown *)
+module TestDb = struct
```

Comment:
Add a warn here these are vulnerable to SQL injection and should never be exposed.